zkevm-circuits/src/tx_circuit/sign_verify.rs

```diff
@@ -416,6 +409,28 @@ impl<F: Field> SignVerifyChip<F> {
         })
     }
 
+    fn assign_word(
+        &self,
+        config: &SignVerifyConfig,
+        ctx: &mut RegionCtx<F>,
+        chips: &ChipsRef<F, NUMBER_OF_LIMBS, BIT_LEN_LIMB>,
+        name: &str,
+        inputs_le: [Value<F>; 32],
```

Comment:
We used to have 
```diff
- inputs_le: [Value<F>; 32],
+ value: Word<F>,
```
and make the lo/hi F handle from caller.

Besides below handling not match `le`. Conventionally we put [0..16] on lo and the other on `hi`

Reply:
fixed [cf7e444](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1418/commits/cf7e444991feb62ad14d93754d25989abd30ef57)